rundeckapp/src/main/groovy/org/rundeck/app/services/EnhancedNodeService.groovy

```diff
@@ -52,7 +52,7 @@ class EnhancedNodeService
 
     @Override
     void afterPropertiesSet() throws Exception {
-        enabled = featureService.featurePresent('enhanced-nodes', false)
+        enabled = featureService.featurePresent('enhancedNodes', false)
```

Comment:
we might want a Constants class for feature names

Reply:
I guess we could have a Features class that has all of the current features. In the pro side we could extend the class like ProFeatures and add the pro only features. What do you think?

Reply:
Yes i agree. probably an enum class would allow us to define both constants and associated mappings when needed.